frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -333,6 +333,11 @@ const plugin: Plugin<ConsumedExtensions> = [
           isList: true,
           kind: models.VirtualMachineInstanceMigrationModel.kind,
         },
+        pvcs: {
```

Comment:
also `virtualMachineConfigurations`. It is a mess: we should fix all of this soon :)

Reply:
Done! And I agree! =)